include/wil/wistd_functional.h

```diff
@@ -262,7 +262,7 @@ namespace wistd     // ("Windows Implementation" std)
         static constexpr size_t __buffer_size = 13 * sizeof(void*);
 
         typedef __function::__base<_Rp(_ArgTypes...)> __base;
-        typename aligned_storage<__buffer_size>::type __buf_;
+        typename aligned_storage<__buffer_size>::type __buf_{};

```

Comment:
IMO this warning should just get suppressed. This has a runtime cost that's entierly unnecessary